book/06-github/sections/1-setting-up-account.asc

```diff
@@ -13,8 +13,11 @@ Go ahead and do this; it's pretty important (as we'll see later).
 
 [NOTE]
 ====
-GitHub provides all of its functionality with free accounts, with the limitation that all of your projects are fully public (everyone has read access).
-GitHub's paid plans also include the option of creating private projects, but we won't be covering those in this book.
+GitHub provides almost all of its functionality with free accounts, except some advanced features.
+Additionally, private repositories are limited to 3 collaborators.
+
+GitHub's paid plans include advanced tools and features as well as increased limits for free services, but we won't be covering those in this book.
+To get more actual information about available plans and their comparison visit https://github.com/pricing[].
```

Comment:
🤔 I don't like the word "actual" here. It's a bit off tone, and it implies that what we've done here is the opposite of provide information. Does this sound better?

```
To get more information about available plans and their comparison, visit https://github.com/pricing[].
```

Reply:
```suggestion
To get more information about available plans and their comparison, visit https://github.com/pricing[].
```